src/collision.cpp

```diff
@@ -0,0 +1,130 @@
+#ifndef COLLISION_HPP
```

Comment:
- Header guards don't belong in c files
- Should go into same file with physics.cpp because physics.cpp is quite short

Reply:
#22